infra/docker/depot/internal/Dockerfile.prod

```diff
@@ -0,0 +1,122 @@
+FROM golang:latest
```

Comment:
I actually wouldn't mind putting an issue up for later and trying to do this if you think it's worth our time in the future?